src/utils/init-fetch.ts

```diff
@@ -0,0 +1,9 @@
+export let isNode = false;
+
+export async function initializeFetch() {
+  if (typeof window === 'undefined' || typeof globalThis.fetch === 'undefined') {
+    const nodeFetch = await import('node-fetch');
+    globalThis.fetch = nodeFetch.default as any;
+    isNode = true;
+  }
```

Comment:
Here you need to take the last PR into account as this code is not well-behaved :) https://github.com/mistralai/client-js/pull/66

Especially you need to export it as a special fetch instead of overriding the global one.

Reply:
have updated the related fetch code~